### DISCUSSION TITLE
Sync brightness between main and additional monitor doesn't work, after sleep or restart.

### DISCUSSION THREAD

**Discussion by lutogin** [MonitorControl/MonitorControl#1385] [Q&A]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Sync brights between main and additional monitor doesn't work, after sleep or restart.
MacBook Pro 2015 macOS 11.7.3 (20G1116) + Dell U2722D
and
MacBook Pro 2019 latest macOS + Dell U2722D

### Steps to reproduce

Just try to use app. It doesn't work at all. Total glitches!

Brightness does what even it wants, random.
Just install your app. And reboot mac. 

![image](https://github.com/MonitorControl/MonitorControl/assets/20394919/9315b443-46db-4997-9249-00be4a967864)

![image](https://github.com/MonitorControl/MonitorControl/assets/20394919/7fe29107-068d-403e-a5b0-2f34bf73968d)

### Expected behavior

Sync brightness work from the main screen. If in room light - brightness increase, if it is darker - brightness decrease. It's very simple.

### Environment Information (please complete the following information)

```markdown
- macOS version: 11.7.3 (20G1116)
- Mac model: MacBook Pro (Retina, 15-inch, Late 2014)
- MonitorControl version: 4.1.0
- Monitor(s): Dell u2722d
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
If you sync brightness from your built-in screen to the external screen and the ambient light sensor (automatic brightness) is enabled, the external display's brightness will change according to the built-in screen. If you use the keyboard to alter the brightness of the external display, the app is trying to keep the delta but if it reaches the min/max end of the range, it can't do that anymore. For a more direct control or if you don't like/understand how syncing works in the app, it's just better to turn it off and change brightness manually as you prefer.

**Comment by iosdevben** [comment]
I was going to say this isn't working for me too, then I had another look at my settings and realised that the Screen to control setting was set to `Depends on mouse pointer position`.

After I changed it to `Change for all screen`, I got the synced behaviour back.

Does that help @lutogin?

Here's a screenshot:
![image](https://github.com/user-attachments/assets/c6a8ec52-5554-4adb-bb38-fb681545b42d)
